src/LoginCallback.tsx

```diff
@@ -30,12 +30,13 @@ const LoginCallback: React.FC<LoginCallbackProps> = ({ errorComponent, loadingEl
       onAuthResume();
       return;
     }
-    oktaAuth.handleLoginRedirect().then(() => {
-      // In `<Security>` component service was not started in case of login redirect.
-      // Start it now after `restoreOriginalUri` has been called and route changed.
-      oktaAuth.start();
```

Comment:
This call is excess because service is started in `<Security>` anyway:
https://github.com/okta/okta-react/blob/4f6f52ff309abcde018eb3d908ec66b1a7f92f2b/src/Security.tsx#L72